File: src/Kpi.Hotels.Dataclient/Context/Models/Order.cs
using System;

namespace Kpi.Hotels.Dataclient.Context.Models
{
    public class Order
    {
        public Guid Id { get; set; }
        public DateTime ArrivalDate { get; set; }
        public DateTime DepartureDate { get; set; }
        public double Total { get; set; }
        public Guid RoomId { get; set; }
        public Room Room { get; set; }
        public Guid ClientId { get; set; }
        public Client Client { get; set; }
    }
}

File: src/Kpi-Hotels-Spa/src/app/models/order.ts
import { client } from './Client';
import { room } from './room';

export	interface order {
	id: any;
	arrivalDate: Date;
	departureDate: Date;
	total: number;
	roomId: any;
	room: room;
	floor: number;
	roomNumber: number;
	fridgeIncluded: boolean;
	clientId: any;
	client: client;
}
